api-audit/src/main/java/com/capitalone/dashboard/evaluator/CodeReviewEvaluatorLegacy.java

```diff
@@ -42,10 +41,10 @@ public CodeReviewEvaluatorLegacy(CommitRepository commitRepository, GitRequestRe
         this.settings = settings;
     }
 
-
     @Override
-    public List<CodeReviewAuditResponse> evaluate(CollectorItem collectorItem, long beginDate, long endDate, Collection data) {
-        return getPeerReviewResponses(collectorItem, beginDate, endDate);
+    public List<CodeReviewAuditResponse> evaluate(CollectorItem collectorItem, List<CollectorItem> collectorItemList,
+                                                  long beginDate, long endDate, Collection<?> data) {
+        return getPeerReviewResponses(collectorItem, collectorItemList, beginDate, endDate);
     }
 
     /**
```

Comment:
I think we should created another interface without changing the existing one.

Reply:
Sure no problem, will create a new one, thanks

Reply:
I have added another overloaded "evaluate" method to the LegacyEvaluator and the Evaluator abstract classes. This worked out well in terms of not disturbing the existing method, and its callers. But if we think we should add another interface only, then I can do that as well, thanks.